### DISCUSSION TITLE
Questions from deepu9

### DISCUSSION THREAD

**Discussion by deepu9** [truflation/trial#2] [General]
I have a few questions:

1. Do you want the data feed for current day or historical data?
2. If historical data, then may I know the range?
3. When you say date_value, do you mean date of the value?
4. Is the value, "Previous close" from the data feed?

Can you let me know. Thanks

**Comment by zolotokrylin** [comment]
1. Historical
2. Range since you start collecting the data
3. Correct
4. What you find is the best fit. Up to you.

**Reply by deepu9** [reply]
Thanks @zolotokrylin

**Reply by zolotokrylin** [reply]
@super0xdev No DMs. Could you read the requirements?

**Reply by zolotokrylin** [reply]
@super0xdev https://github.com/truflation/trial?tab=readme-ov-file#questions

**Reply by zolotokrylin** [reply]
you PR https://github.com/truflation/trial/pull/5 is in review, @super0xdev
